hibernate-core/src/main/java/org/hibernate/type/format/jackson/Jackson3JsonFormatMapper.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate.type.format.jackson;
+
+import org.hibernate.type.descriptor.WrapperOptions;
+import org.hibernate.type.descriptor.java.JavaType;
+import org.hibernate.type.format.AbstractJsonFormatMapper;
+import org.hibernate.type.format.FormatMapperCreationContext;
+import tools.jackson.core.JacksonException;
+import tools.jackson.core.JsonGenerator;
+import tools.jackson.core.JsonParser;
+import tools.jackson.databind.JacksonModule;
+import tools.jackson.databind.cfg.MapperBuilder;
+import tools.jackson.databind.json.JsonMapper;
+
+import java.lang.reflect.Type;
+import java.util.List;
+
+/**
+ * @author Christian Beikov
+ * @author Yanming Zhou
+ * @author Nick Rayburn
+ */
```

Comment:
Not sure if the previous author's need to be retained. This file was copied from the existing one that they were on. (I also don't need my name in here either unless it's the project standard, just tried to match what I copied from.)